docsets/Envoy/docset.json

```diff
@@ -1,12 +1,28 @@
 {
     "name": "Envoy",
-    "version": "1.17.1",
+    "version": "1.21.1",
     "archive": "Envoy.tgz",
     "author": {
         "name": "Ruoshan Huang",
         "link": "https://twitter.com/_ruoshan"
     },
     "specific_versions": [
+        {
+            "version": "1.21.1",
+            "archive": "versions/1.21.1/Envoy.tgz"
+        },
+        {
+            "version": "1.20.1",
+            "archive": "versions/1.20.1/Envoy.tgz"
+        },
+        {
+            "version": "1.19.1",
+            "archive": "versions/1.19.1/Envoy.tgz"
+        },
+        {
+            "version": "1.18.4",
+            "archive": "versions/1.18.4/Envoy.tgz"
```

Comment:
The archive at this path does not exist. You added `versions/v1.18.4/Envoy.tgz` instead.